FirstProject/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
Add `.idea` to this file and you should be able to safely clean up your PR from IDE-specific files, while still keeping them around for local development.